docs/octant-plugin-installation.md

```diff
@@ -91,18 +91,22 @@ You can follow the steps listed below to install octant and antrea-octant-plugin
     export KUBECONFIG=/etc/kubernetes/admin.conf
     ```
 
-3. Build antrea-octant-plugin.
+3. Get corresponding antrea-octant-plugin binary from release/assets based on your environment.
 
+    For example, you can get antrea-octant-plugin-linux-x86_64 if it meets your operating system  and architecture.
+    Then, rename the binary to antrea-octant-plugin so that you do not need to change any command for step4.
```

Comment:
s/step4/step 4
But I think we can merge step 3 and 4 to make the instructions less verbose:
```
wget -O antrea-octant-plugin https://github.com/vmware-tanzu/antrea/releases/download/v0.8.0/antrea-octant-plugin-linux-x86_64
chmod a+x antrea-octant-plugin
mv antrea-octant-plugin $HOME/.config/octant/plugins/
```

Reply:
Thanks for the suggestion. Have updated accordingly.